relayer/txmgr/config.go

```diff
@@ -68,7 +68,7 @@ func NewCLIConfig(rpc string, defaults DefaultFlagValues) CLIConfig {
 		NetworkTimeout:            defaults.NetworkTimeout,
 		TxSendTimeout:             defaults.TxSendTimeout,
 		TxNotInMempoolTimeout:     defaults.TxNotInMempoolTimeout,
-		ReceiptQueryInterval:      defaults.ReceiptQueryInterval,
+		ReceiptQueryInterval:      receiptQueryInterval,
```

Comment:
[Variable names in Go should be short rather than long.
](https://go.dev/wiki/CodeReviewComments#variable-names)
`receiptQueryInterval` is an unnecessarily long var name here. It's only ever used once in a field assignment so a much shorter name should be used. A more appropriate name would be only one word or a few character long.